gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -464,28 +464,34 @@ MM_MemorySubSpaceSemiSpace::tearDown(MM_EnvironmentBase *env)
 void
 MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 {
+	Trc_MM_MSSSS_flip_entry(env->getLanguageVMThread());
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
 	bool debug = _extensions->debugTiltedScavenge;
 #endif
 	switch (step) {
 	case set_evacuate:
+		Trc_MM_MSSSS_flip_set_evacuate(env->getLanguageVMThread());
```

Comment:
you can share a few of those 'set' ones by expanding by a string parameter and you pass respectively " evacuate", " allocate",...